boxoffice/views/admin_report.py

```diff
@@ -137,3 +148,23 @@ def row_handler(row):
         csv_headers.remove('attendee_details')
 
     return csv_response(csv_headers, rows, row_type='dict', row_handler=row_handler)
+
+@app.route('/admin/o/<org_name>/invoices.csv')
+@lastuser.requires_login
+@load_models(
+    (Organization, {'name': 'org_name'}, 'organization'),
+    permission='org_admin')
+def invoices_report(organization):
+    headers, rows = organization.fetch_invoices()
+
+    def row_handler(row):
+        dict_row = {}
+        for idx, item in enumerate(row):
+            dict_row[headers[idx]] = item
```

Comment:
Reduce these three lines to `dict(zip(headers, row))`. [Zip's documentation](https://docs.python.org/2/library/functions.html#zip).